keps/sig-network/20200103-mixed-protocol-lb.md

```diff
@@ -351,9 +351,35 @@ In the first release:
  - indicate clearly to the user what ports with what protocols have been opened on the LB
  - preferably not create any Cloud LB resources if the Service definition contains unsupported protocols.
  
- In order to provide a way for the CPI to indicate port statuses to the user we would add the following new `portStatus` list of port statuses to `Service.status.loadBalancer`, so the CPI can indicate the status of the LB:
+ In order to provide a way for the CPI to indicate port statuses to the user we would add the following new `portStatus` list of port statuses to `Service.status.loadBalancer.ingress`, so the CPI can indicate the status of the LB. Also we add a `conditions` list of conditions to the `Service.status.loadBalancer`, with the first official condition `LoadBalancerMixedProtocolNotSupported` defined.
 
 ```json
+"io.k8s.api.core.v1.LoadBalancerCondition": {
```

Comment:
I have to say, it's considerably harder to review this as openAPI than as Go code :)

Reply:
Yep, it sounded like a good choice for API specification. 😄 +1 that Go would be easier 😄